lrauv_ignition_plugins/src/TethysCommPlugin.cc

```diff
@@ -412,21 +412,8 @@ void TethysCommPlugin::CommandCallback(
 
   // Thruster
   ignition::msgs::Double thrusterMsg;
-  // TODO(arjo):
-  // Conversion from angular velocity to force b/c thruster plugin takes force
-  // Maybe we should change that?
-  // https://github.com/osrf/lrauv/issues/75
   auto angVel = _msg.propomegaaction_();
-
-  // force = thrust_coefficient * fluid_density * omega ^ 2 *
-  //         propeller_diameter ^ 4
-  // These values are defined in the model's Thruster plugin's SDF
-  auto force = 0.004422 * 1000 * pow(0.2, 4) * angVel * angVel;
-  if (angVel < 0)
-  {
-    force *=-1;
-  }
-  thrusterMsg.set_data(force);
+  thrusterMsg.set_data(angVel);
   this->thrusterPub.Publish(thrusterMsg);
```

Comment:
We should also change `thrusterTopic` to `cmd_vel`, otherwise it's treating this as angular velocity

```diff
diff --git a/lrauv_ignition_plugins/src/TethysCommPlugin.hh b/lrauv_ignition_plugins/src/TethysCommPlugin.hh
index 1b18ae4..8eecae4 100644
--- a/lrauv_ignition_plugins/src/TethysCommPlugin.hh
+++ b/lrauv_ignition_plugins/src/TethysCommPlugin.hh
@@ -106,7 +106,7 @@ namespace tethys
 
     /// Topic to publish to for thruster
     private: std::string thrusterTopic
-      {"propeller_joint/cmd_pos"};
+      {"propeller_joint/cmd_vel"};
 
     /// Topic to publish to for rudder
     private: std::string rudderTopic
```